mobile_files/package.json.orig

```diff
@@ -61,6 +61,7 @@
     "react-native-udp": "git+https://github.com/status-im/react-native-udp.git#2.3.1-1",
     "react-native-webview-bridge": "git+https://github.com/status-im/react-native-webview-bridge.git#0.33.16-4",
     "react-navigation": "^2.12.1",
+    "react-navigation-stack": "git+https://github.com/status-im/react-navigation-stack#6d1133b25c4d6ba059cffe7ccf69c92da628a448",
```

Comment:
what is this for?

Reply:
https://github.com/status-im/react-navigation-stack/commit/6d1133b25c4d6ba059cffe7ccf69c92da628a448

Reply:
ok I see it is a transitive dependency of react-navigation
probably worth pushing upstream if we stick with it

Reply:
nice catch!

Reply:
we need to upgrade react-navigation in out project first